feature-libs/asm/components/asm-session-timer/asm-session-timer.component.ts

```diff
@@ -73,6 +73,11 @@ export class AsmSessionTimerComponent implements OnInit, OnDestroy {
   }
 
   private getTimerStartDelayInSeconds(): number {
+    if (
+      this.config?.asm?.agentSessionTimer?.startingDelayInSeconds === undefined
+    ) {
+      return 600;
+    }
```

Comment:
Comment for reviewer: I wonder if we will need to manage undefined for all the configuration values.  Perhaps there is a better way?

Reply:
Fixed.  Dropped `?` after `this.config`